frontend/stylesheets/main.less

```
@@ -174,7 +174,8 @@ body {
     background: #f6ff2e9c;
   }
 
-  100% {}
+  100% {
+  }
```

Stylelint (17.9.0)

[error] Empty block (block-no-empty)

(block-no-empty)